modules/ROOT/pages/errors/gql-errors.adoc

```diff
@@ -946,10 +991,36 @@ Status description:: error: syntax error or access rule violation - unsupported
 
 Status description:: error: syntax error or access rule violation - invalid parameter. Invalid parameter `{ $param }`.
 
+=== 42N53
+
+Status description:: error: syntax error or access rule violation - unsafe usage of repeatable elements. The quantified path pattern may yield an infinite number of rows under match mode `REPEATABLE ELEMENTS`. Use a path selector or add an upper bound to the quantified path pattern.
```

Comment:
```suggestion
Status description:: error: syntax error or access rule violation - unsafe usage of repeatable elements. The quantified path pattern may yield an infinite number of rows under match mode `REPEATABLE ELEMENTS'. Add an upper bound to the quantified path pattern.
```

Reply:
Doesn't this become wrong because now again REPEATABLE ELEMENTS have different quote before and after. I'm fine with either quoting for now but if they are different I suspect it will screw up docs formatting

Reply:
My bad. I fixed backticks